src/RoleObjects/ComboBoxData.test.js

```diff
@@ -5,6 +5,7 @@ import { parentEl, stage, container } from '../__jestSharedSetup';
 describe('ComboBoxData', () => {
   describe('register role', () => {
     let cjsComboBox;
+    let cjsSpan;
```

Comment:
so can we combine all the variables under single "let". In all the files below.
`let cjsComboBox, cjsSpan, comboBoxEl, isExpanded, isReadOnly, isRequired, shouldAutoComplete;`

Reply:
It's easier for diffs to keep them separate.